go.mod

```diff
@@ -1,8 +1,6 @@
 module k8s.io/minikube
 
-go 1.23.0
-
-toolchain go1.23.4
```

Comment:
would it worth to use toolchain local?

Reply:
i've also added setting `GOTOOLCHAIN` to `local` in `Makefile`